migrations/Version202112231326462260_taoQtiTest.php

```diff
@@ -0,0 +1,114 @@
+use oat\taoQtiTest\models\event\ResultItemVariablesTransmissionEvent;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version202112231326462260_taoQtiTest extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        $eventManager = $this->getEventManager();
+        $this->attachResultItemVariablesTransmissionEvent($eventManager);
+        $this->attachResultTestVariablesTransmissionEvent($eventManager);
+        $this->getServiceManager()->register(EventManager::SERVICE_ID, $eventManager);
+    }
+
+    public function down(Schema $schema): void
+    {
+        $eventManager = $this->getEventManager();
+        $this->detachResultItemVariablesTransmissionEvent($eventManager);
+        $this->detachResultTestVariablesTransmissionEvent($eventManager);
+        $this->getServiceManager()->register(EventManager::SERVICE_ID, $eventManager);
+    }
+
+    private function getEventManager(): EventManager
+    {
+        return $this->getServiceLocator()->get(EventManager::class);
+    }
+
+    private function attachResultItemVariablesTransmissionEvent(EventManager $eventManager): void
+    {
+        $eventManager->attach(
+            ResultItemVariablesTransmissionEvent::class,
+            [ResultTransmissionEventHandlerInterface::SERVICE_ID, 'transmitResultItemVariable']
+        );
+
+        $this->addReport(
+            Report::createSuccess(
+                sprintf(
+                    'Unsubscribed %s::%s to %s event',
+                    ResultTransmissionEventHandlerInterface::class,
+                    'transmitResultItemVariable',
+                    ResultItemVariablesTransmissionEvent::class
+                )
+            )
+        );
+    }
+
+    private function detachResultItemVariablesTransmissionEvent(EventManager $eventManager): void
+    {
+        $eventManager->detach(
+            ResultItemVariablesTransmissionEvent::class,
+            [ResultTransmissionEventHandlerInterface::SERVICE_ID, 'transmitResultItemVariable']
+        );
+        $this->addReport(
+            Report::createSuccess(
+                sprintf(
+                    'Unsubscribed %s::%s to %s event',
+                    ResultTransmissionEventHandlerInterface::class,
+                    'transmitResultItemVariable',
+                    ResultItemVariablesTransmissionEvent::class
+                )
+            )
+        );
+    }
+
+    private function attachResultTestVariablesTransmissionEvent(EventManager $eventManager): void
+    {
+        $eventManager->attach(
+            ResultTestVariablesTransmissionEvent::class,
+            [ResultTransmissionEventHandlerInterface::SERVICE_ID, 'transmitResultTestVariable']
+        );
+
+        $this->addReport(
+            Report::createSuccess(
+                sprintf(
+                    'Unsubscribed %s::%s to %s event',
```

Comment:
Shouldn't be like that?

```suggestion
                    'Subscribed %s::%s to %s event',
```